src/components/molecules/TitleWIthAvatar/index.js

```diff
@@ -22,7 +22,7 @@ export default function TitleWithAvatar(props) {
   return (
     <div className={classes.container}>
       <CustomAvatar {...props} />
-      <Typography className={classes.by}> by {props.author}</Typography>
+      <Typography className={classes.by}>{props.title}</Typography>
```

Comment:
Why do we have Title instead of Author? Won't that confuse with the Post Title?

Reply:
Thanks for the comment abdul,
but, as we are at molecule level (TitleWithAvatar), which contains title and the avatar. so, as per the name of the molecule i have changed the author to avatar.